### DISCUSSION TITLE
What would we like to use this repo for? (Brainstorming thread)

### DISCUSSION THREAD

**Discussion by erikgustafsson** [Nordic-Accessibility-Community-Group/en-301-549-resources-and-eaa-monitoring#7] [Ideas]
Please add ideas of how you want to use this repo.

- Additional content.
- Ways of working with the repo.
- Links to inspirational ideas.
- ...and so on.

**Comment by sander-nl** [comment]
Trying to get consensus on some of the grey areas in interpreting success criteria.
